plugins/org.jkiss.dbeaver.ui.editors.session/src/org/jkiss/dbeaver/ui/views/session/SessionManagerViewer.java

```diff
@@ -215,7 +215,7 @@ public void widgetSelected(SelectionEvent e) {
                                 DetailsListControl detailsProps = new DetailsListControl(detailsFolder, workbenchPart.getSite(), detailsInfo);
                                 extDetailsItem.setControl(detailsProps);
                             } else {
-                                extDetailsItem.setControl(UIUtils.createLabel(detailsFolder, "Unsupported details type: " + detailsType));
+                                extDetailsItem.setControl(UIUtils.createLabel(detailsFolder, SessionEditorMessages.viewer_unsupported_details_type_label + " " + detailsType));
```

Comment:
Let's embed the count into the localization string to avoid weird concatenations.